packages/libs/client-utils/src/nodejs/test/aux-files/real-tx-tpl.yaml

```diff
@@ -0,0 +1,27 @@
+# A Kadena gas station is nothing more than a KDA account controlled by a
+# special guard. This transaction template constructs gas station
+# safely...i.e. it calls rotate after creating the account to ensure that you
+# will be able to rotate it's guard in the future if any of the gas station's
+# limits need to be changed. The mk-guard lambda function makes the important
+# parts of the guard DRY to minimize the chance of mistakes.
+codeFile: ./aux-files/real-tx-tpl-code.pact
+data:
+meta:
+  chainId: '{{chain}}'
+  sender: '{{{funding-acct}}}'
+  gasLimit: 2000
+  gasPrice: 0.00000001
+  ttl: 7200
+networkId: '{{network}}'
+signers:
+  - public: '{{funding-key}}'
+    caps:
+      - name: 'coin.TRANSFER'
+        args: ['{{{funding-acct}}}', '{{{gas-station-name}}}', { { amount } }]
```

Comment:
Are the spaces on purpose? `{ { amount } }`

Reply:
Auto-formatting did this. Fixed 👍🏽